app/views/anonymous_feedback/_feedback.html.erb

```diff
@@ -1,7 +1,14 @@
 <% if anonymous_contact.is_a?(Support::Requests::Anonymous::ProblemReport) %>
-  <strong>action</strong>: <%= anonymous_contact.what_doing %>
-  <br/>
-  <strong>problem</strong>: <%= anonymous_contact.what_wrong %>
+  <% if anonymous_contact.what_doing %>
```

Comment:
You might want to do `if anonymous_contact.what_doing.present?` here to guard against `what_doing` being an empty string.